### PR TITLE
Polish typography

### PR DESCRIPTION
**Pull request by PiotrekD145**
I manually replaced all American quotation marks in the Polish texts with their Polish equivalents.

### CODE REVIEW

**Comment by flashcode** [comment]
Hi,
There's an invalid control sequence on line 9130 (an extra `\`), could you please fix?
Thanks.

**Comment by PiotrekD145** [comment]
Oh, _mea culpa_. It should be fixed now.

**Comment by flashcode** [comment]
Only the end of file is affected, not all translations?

**Comment by PiotrekD145** [comment]
Hmmm… That's really strange, I remember I fixed everything from the beginning to the end, but now I see only the end is fixed… I'll fix the first part soon, when I got some time. Could you wait a few days, until I do that, please?

**Comment by flashcode** [comment]
Sure.

**Comment by flashcode** [comment]
Hi @PiotrekD145, some news about this PR?

**Comment by PiotrekD145** [comment]
Oh, I completely forgot about the thing, sorry. I'll try to take care of it during the Christmas time.

**Comment by PiotrekD145** [comment]
@flashcode: OK, so I finally did it. It should be fine.

File: po/pl.po [hunk 1]
```diff
@@ -9195,62 +9195,62 @@ msgstr "%s%s[%s%s%s] %s%s%s uciszony"
 
 #, c-format
 msgid "%s%s: command \"%s\" not found:"
-msgstr "%s%s: nie znaleziono komendy \"%s\":"
+msgstr "%s%s: nie znaleziono komendy „%s\”:"
```
Comment (flashcode) [inline po/pl.po]: There's a trailing `\` here (detected by automatic tests on Travis: https://travis-ci.org/weechat/weechat/jobs/187174935#L1050).
Could you please fix that please?
Thanks.

**Comment by flashcode** [comment]
@soltysek: any opinion on this? (see the PR changes and my comment above)

**Comment by soltysek** [comment]
As far as I can see not all quotation marks have been changed (in some strings only first quotation mark is changed). In my opinion @flashcode is right about that not all quotation marks should be converted to 
Polish. I'll try to look into this PR tomorrow and give more feedback.

File: po/pl.po [hunk 2]
```diff
@@ -5438,16 +5438,16 @@ msgid ""
 "buffer pointer (\"0x12345678\") or buffer full name (\"irc.freenode.#weechat"
 "\")"
 msgstr ""
-"wskaźnik na bufor (\"0x12345678\") lub jego pełna nazwa (\"irc.freenode."
-"#weechat\")"
+"wskaźnik na bufor („0x12345678”) lub jego pełna nazwa (\"irc.freenode."
```
Comment (soltysek) [inline po/pl.po]: quotation mark before _irc.freenode._ hould be also changed

File: po/pl.po [hunk 3]
```diff
@@ -5483,7 +5483,7 @@ msgstr ""
 
 #, c-format
 msgid "%s: %s, \"%s\": removed"
-msgstr "%s: %s, \"%s\": usunięto"
+msgstr "%s: %s, „%s\": usunięto"
```
Comment (soltysek) [inline po/pl.po]: quotation mark after third _%s_ should also be changed

File: po/pl.po [hunk 4]
```diff
@@ -5691,8 +5691,8 @@ msgstr ""
 "in/-inclose)\n"
 "-nostdin: nie twórz strumienia dla stdin (domyślne)\n"
 " -buffer: wyświetl/wyślij wyjście komendy do tego bufora (jeśli bufor nie "
-"zostanie znaleziony zostanie utworzony nowy bufor o nazwie \"exec.exec.xxx"
-"\")\n"
+"zostanie znaleziony zostanie utworzony nowy bufor o nazwie „exec.exec.xxx"
+"„)\n"
```
Comment (soltysek) [inline po/pl.po]: this should be upper quotation mark

File: po/pl.po [hunk 5]
```diff
@@ -5751,7 +5751,7 @@ msgstr ""
 "  /exec -nf -noln links -dump https://weechat.org/files/doc/devel/"
 "weechat_user.en.html\n"
 "  /exec -o uptime\n"
-"  /exec -pipe \"/print Machine uptime:\" uptime\n"
+"  /exec -pipe „/print Machine uptime:” uptime\n"
```
Comment (soltysek) [inline po/pl.po]: quotation marks in this line shouldn't be changed

File: po/pl.po [hunk 6]
```diff
@@ -6227,15 +6227,15 @@ msgstr "Brak serwerów"
 
 #, c-format
 msgid "Servers with \"%s\":"
-msgstr "Serwery z \"%s\":"
+msgstr "Serwery z „%s\":"
```
Comment (soltysek) [inline po/pl.po]: quotation mark after _%s_ should also be changed

File: po/pl.po [hunk 7]
```diff
@@ -6614,9 +6614,9 @@ msgstr ""
 "plik: nazwa pliku (na lokalnym hoście)\n"
 "\n"
 "Przykłady:\n"
-"  rozmowa z użytkownikiem \"toto\":\n"
+"  rozmowa z użytkownikiem „toto\":\n"
```
Comment (soltysek) [inline po/pl.po]: quotation mark after _toto_ should also be changed

File: po/pl.po [hunk 8]
```diff
@@ -6614,9 +6614,9 @@ msgstr ""
...
 "    /dcc chat toto\n"
-"  wyśle plik \"/home/foo/bar.txt\" użytkownikowi \"toto\":\n"
+"  wyśle plik „/home/foo/bar.txt” użytkownikowi „toto\":\n"
```
Comment (soltysek) [inline po/pl.po]: quotation mark after _toto_ should also be changed

File: po/pl.po [hunk 9]
```diff
@@ -6718,24 +6718,24 @@ msgid ""
...
 "   serwer: wewnętrzna nazwa serwera, na którym dana osoba ma być ignorowana\n"
 "  kanał: nazwa kanału, na którym dana osoba ma być ignorowana\n"
 "\n"
-"Uwaga: wyrażenie regularne może zaczynać się od \"(?-i)\" jeśli wielkość "
+"Uwaga: wyrażenie regularne może zaczynać się od „(?-i)” jeśli wielkość "
 "znaków ma mieć znaczenie.\n"
 "\n"
 "Przykłady:\n"
-"  ignoruje wszędzie nick \"toto\":\n"
+"  ignoruje wszędzie nick „toto\":\n"
```
Comment (soltysek) [inline po/pl.po]: quotation mark after _toto_ should also be changed

File: po/pl.po [hunk 10]
```diff
@@ -6834,10 +6834,10 @@ msgstr ""
 "podmieniane na odpowiednie wartości)\n"
 "\n"
 "Możliwe jest kopanie/banowanie za pomocą maski, nick zostanie wyciągnięty z "
-"maski i zastąpiony \"*\".\n"
+"maski i zastąpiony „*”.\n"
 "\n"
 "Przykład:\n"
-"  zbanuje \"*!*@host.com\", następnie wykopie \"toto\":\n"
+"  zbanuje „*!*@host.com”, następnie wykopie „toto\":\n"
```
Comment (soltysek) [inline po/pl.po]: quotation mark after _toto_ should also be changed

File: po/pl.po [hunk 11]
```diff
@@ -7745,17 +7745,17 @@ msgid ""
...
 msgid ""
 "string with priorities for gnutls (for syntax, see documentation of function "
 "gnutls_priority_init in gnutls manual, common strings are: \"PERFORMANCE\", "
 "\"NORMAL\", \"SECURE128\", \"SECURE256\", \"EXPORT\", \"NONE\")"
 msgstr ""
 "ciąg z priorytetami dla gnutls (składnię można znaleźć w dokumentacji gnutls "
-"dla funkcji  gnutls_priority_init, często używane ciągi to: \"PERFORMANCE\", "
-"\"NORMAL\", \"SECURE128\", \"SECURE256\", \"EXPORT\", \"NONE\")"
+"dla funkcji  gnutls_priority_init, często używane ciągi to: „PERFORMANCE”, "
+"„NORMAL”, „SECURE128\", „SECURE256\", „EXPORT\", „NONE”)"
```
Comment (soltysek) [inline po/pl.po]: quotation marks after SECURE128, SECURE256, EXPORT should be also changed

File: po/pl.po [hunk 12]
```diff
@@ -7790,7 +7790,7 @@ msgid ""
 msgstr ""
 "oddzielona przecinkami lista opcji klienta do włączenia dla serwera, jeśli "
 "są dostępne  (zobacz /help cap żeby poznać listę opcji wspieranych przez "
-"WeeChat) (przykład: \"away-notify,multi-prefix\")"
+"WeeChat) (przykład: „away-notify,multi-prefix”)"
```
Comment (soltysek) [inline po/pl.po]: quotation marks in this line shouldn't be changed

File: po/pl.po [hunk 13]
```diff
@@ -7935,8 +7935,8 @@ msgstr ""
 "oddzielona przecinkami lista kanałów do wejścia po połączeniu z serwerem (po "
 "wykonaniu komendy + opóźnienie jeśli są ustawione); kanały wymagające hasła "
 "muszą znajdować się na początku listy, wszystkie hasła muszą zostać podane "
-"po kanałach (oddzielone spacją) (przykład: \"#kanał1,#kanał2,#kanał3 hasło1,"
-"hasło2\", gdzie #kanał1 i #kanał2 wymagają odpowiednio hasło1 i hasło2) "
+"po kanałach (oddzielone spacją) (przykład: „#kanał1,#kanał2,#kanał3 hasło1,"
```
Comment (soltysek) [inline po/pl.po]: quotation marks in this line and line below shouldn't be changed

File: po/pl.po [hunk 14]
```diff
@@ -8392,19 +8392,19 @@ msgid ""
 "examples: \"ovh\": filter modes o/v/h, \"-bkl\": filter all modes but not b/"
 "k/l"
 msgstr ""
-"włącz inteligentny filtr dla wiadomości \"mode\": \"*\" filtruje wszystkie "
-"tryby, \"+\" filtruje wszystkie tryby w prefiksach serwera (na przykład \"ovh"
-"\"), \"xyz\" filtruje tylko tryby x/y/z, \"-xyz\" filtruje wszystkie tryby "
-"poza x/y/z; przykład: \"ovh\": filtruje tryby o/v/h, \"-bkl\" filtruje "
+"włącz inteligentny filtr dla wiadomości „mode\": „*” filtruje wszystkie "
```
Comment (soltysek) [inline po/pl.po]: quotation mark after _mode_ should also be changed

File: po/pl.po [hunk 15]
```diff
@@ -8392,19 +8392,19 @@ msgid ""
...
+"tryby, „+” filtruje wszystkie tryby w prefiksach serwera (na przykład „ovh"
+"„), „xyz” filtruje tylko tryby x/y/z, „-xyz” filtruje wszystkie tryby "
```
Comment (soltysek) [inline po/pl.po]: first quotation mark in this line should be upper

File: po/pl.po [hunk 16]
```diff
@@ -8392,19 +8392,19 @@ msgid ""
...
+"poza x/y/z; przykład: „ovh\": filtruje tryby o/v/h, „-bkl” filtruje "
```
Comment (soltysek) [inline po/pl.po]: quotation mark after _ovh_ should also be changed

File: po/pl.po [hunk 17]
```diff
@@ -8448,11 +8448,11 @@ msgid ""
...
-"zamień kolory mirca w wiadomościach używając tablicy hashy: klucze to  \"fg,"
-"bg\" przyjmują one wartości pomiędzy -1 (nie określono) a 15, wartości to "
-"nazwy lub numery kolorów WeeChat (format: \"1,-1:kolor1;2,7:kolor2\"), "
-"przykład: \"1,-1:darkgray;1,2:white,blue\" w celu przemapowania czarnego na "
-"\"darkgray\" oraz czarnego na niebieskim tle na \"white,blue\"; domyślne "
+"zamień kolory mirca w wiadomościach używając tablicy hashy: klucze to  „fg,"
+"bg” przyjmują one wartości pomiędzy -1 (nie określono) a 15, wartości to "
+"nazwy lub numery kolorów WeeChat (format: „1,-1:kolor1;2,7:kolor2”), "
+"przykład: „1,-1:darkgray;1,2:white,blue” w celu przemapowania czarnego na "
+"„darkgray” oraz czarnego na niebieskim tle na „white,blue\"; domyślne "
```
Comment (soltysek) [inline po/pl.po]: quotation mark after _blue_ should also be changed

File: po/pl.po [hunk 18]
```diff
@@ -8448,11 +8448,11 @@ msgid ""
...
 msgstr ""
-"zamień kolory mirca w wiadomościach używając tablicy hashy: klucze to  \"fg,"
-"bg\" przyjmują one wartości pomiędzy -1 (nie określono) a 15, wartości to "
-"nazwy lub numery kolorów WeeChat (format: \"1,-1:kolor1;2,7:kolor2\"), "
-"przykład: \"1,-1:darkgray;1,2:white,blue\" w celu przemapowania czarnego na "
-"\"darkgray\" oraz czarnego na niebieskim tle na \"white,blue\"; domyślne "
+"zamień kolory mirca w wiadomościach używając tablicy hashy: klucze to  „fg,"
+"bg” przyjmują one wartości pomiędzy -1 (nie określono) a 15, wartości to "
+"nazwy lub numery kolorów WeeChat (format: „1,-1:kolor1;2,7:kolor2”), "
+"przykład: „1,-1:darkgray;1,2:white,blue” w celu przemapowania czarnego na "
```
Comment (soltysek) [inline po/pl.po]: quotation marks in this line shouldn't be changed

File: po/pl.po [hunk 19]
```diff
@@ -8507,8 +8507,8 @@ msgid ""
 msgstr ""
 "domyślna maska dla bana dla komend /ban, /unban i /kickban; zmienne $nick, "
 "$user, $ident i $host są zastępowane odpowiednimi wartościami (pobranymi z "
-"\"nick!user@host\"); $ident jest taki sam jak $user, jeśli $user nie zaczyna "
-"się od \"~\", w przeciwnym wypadku jest ustawiany na \"*\"; domyślna maska "
+"„nick!user@host”); $ident jest taki sam jak $user, jeśli $user nie zaczyna "
+"się od „~”, w przeciwnym wypadku jest ustawiany na „*\"; domyślna maska "
```
Comment (soltysek) [inline po/pl.po]: quotation mark after _*_ should also be changed

File: po/pl.po [hunk 20]
```diff
@@ -8696,7 +8696,7 @@ msgid "parse an IRC message"
 msgstr "przetwarza wiadomość IRC"
 
 msgid "\"message\": IRC message, \"server\": server name (optional)"
-msgstr "\"message\": wiadomość IRC, \"server\": nazwa serwera (opcjonalne)"
+msgstr "\"message\": wiadomość IRC, „server\": nazwa serwera (opcjonalne)"
```
Comment (soltysek) [inline po/pl.po]: quotation mark after _server_ should also be changed

File: po/pl.po [hunk 21]
```diff
@@ -8709,15 +8709,15 @@ msgid ""
...
-"\" (\"-1\" jeśli \"text\" nie został znaleziony)"
+"„tags\": tagi, „message_without_tags\": wiadomość bez tagów, „nick\": "
+"nick, „host\": host, „command\": komenda, „channel\": kanał, „arguments"
+"„: argumenty (razem z kanałem), „text\": tekst (na przykład wiadomość "
+"użytkownika), „pos_command\": indeks wiadomości „command” message („-1” "
+"jeśli „command” nie zostało znalezione), „pos_arguments\": indeks "
+"wiadomości „arguments” („-1” jeśli „arguments” nie zostało "
+"znalezione), „pos_channel\": indeks wiadomości „channel” („-1” jeśli "
+"„channel” nie został znaleziony), „pos_text\": indeks słowa „text"
+"” („-1” jeśli „text” nie został znaleziony)"
```
Comment (soltysek) [inline po/pl.po]: not all quotation marks in above section was changed, please fix it

File: po/pl.po [hunk 22]
```diff
@@ -8727,8 +8727,8 @@ msgid ""
 "\"msg1\" ... \"msgN\": messages to send (without final \"\\r\\n\"), "
 "\"args1\" ... \"argsN\": arguments of messages, \"count\": number of messages"
 msgstr ""
-"\"msg1\" ... \"msgN\": wiadomości do wysłania (bez kończącego \"\\r\\n\"), "
-"\"args1\" ... \"argsN\": argumenty wiadomości, \"count\": ilość wiadomości"
+"„msg1” ... „msgN\": wiadomości do wysłania (bez kończącego „\\r\\n”), "
+"„args1” ... „argsN\": argumenty wiadomości, „count\": ilość wiadomości"
```
Comment (soltysek) [inline po/pl.po]: not all quotation marks in above section was changed, please fix it

File: po/pl.po [hunk 23]
```diff
@@ -9686,7 +9686,7 @@ msgstr ""
 "    /set logger.level.core.weechat 0\n"
 "  użyj oddzielnych katalogów dla serwerów IRC, oraz oddzielnych plików dla "
 "kanałów:\n"
-"    /set logger.mask.irc \"$server/$channel.weechatlog\""
+"    /set logger.mask.irc „$server/$channel.weechatlog”"
```
Comment (soltysek) [inline po/pl.po]: quotation marks in this line shouldn't be changed

File: po/pl.po [hunk 24]
```diff
@@ -10573,22 +10573,22 @@ msgstr ""
...
 "połączyć się na ustawionym porcie.\n"
-"Protokół \"weechat\" pozwala na połączenie się zdalnym interfejsem, lista "
+"Protokół „weechat” pozwala na połączenie się zdalnym interfejsem, lista "
 "dostępna pod adresem: https://weechat.org/download/\n"
 "\n"
 "Bez argumentu komenda otwiera bufor z listą klientów.\n"
 "\n"
 "Przykłady:\n"
-"  pośrednik irc dla serwera \"freenode\":\n"
+"  pośrednik irc dla serwera „freenode\":\n"
```
Comment (soltysek) [inline po/pl.po]: quotation mark after _freenode_ should also be changed

File: po/pl.po [hunk 25]
```diff
@@ -11895,19 +11895,19 @@ msgstr ""
...
+"    /trigger add resize_small signal signal_sigwinch „${info:term_width} < "
+"100” „” „/bar hide nicklist”\n"
+"    /trigger add resize_big signal signal_sigwinch „${info:term_width} >= "
+"100” „” „/bar show nicklist”\n"
 "  cichy zapis plików konfiguracyjnych co każdą godzinę:\n"
-"    /trigger add cfgsave timer 3600000;0;0 \"\" \"\" \"/mute /save\"\n"
+"    /trigger add cfgsave timer 3600000;0;0 „” „” „/mute /save”\n"
 "  otwiera monitor triggerów i pokazuje tylko modifiers i triggery, których "
-"nazwa zaczyna się od \"resize\":\n"
+"nazwa zaczyna się od „resize\":\n"
```
Comment (soltysek) [inline po/pl.po]: quotation marks in example command shouldn't be changed

**Comment by PiotrekD145** [comment]
OK, I fixed those and a few more.

**Comment by PiotrekD145** [comment]
Should be fixed unless I have broken something.

**Comment by flashcode** [comment]
Thanks for the changes.
If I merge this PR, many translations (about 130) would be lost.
Maybe you could rebase your changes or reapply them on the current master branch?
Thank you.

**Comment by soltysek** [comment]
Please also check if all quotation marks are proper replaced. As I remember number of opening and closing quotation marks wasn't the same.

**Comment by flashcode** [comment]
Closing this old PR.
If needed, please submit an new one without errors, and updated on the latest codebase.

**Pull request closed without merging**